### DISCUSSION TITLE
Misiones bug en Felwood

### DISCUSSION THREAD

**Discussion by Imirrow** [chromiecraft/chromiecraft#2577] [General]
Saludos, encontré una serie de misiones bug en Felwood son las siguientes:

- Flor de viento Corrupta
- Blancoria Corrupta
- Melodia Corrupta
- Dragon nocturno Corrupta

**Comment by Shineslippers** [comment]
That is known issues and it's already fixed. Anyway place for bug reports is there `https://github.com/chromiecraft/chromiecraft/issues`
